dev-docs/source/ObtainingABenchmarkForMantidFitting.rst

```diff
@@ -0,0 +1,91 @@
+
+This page will explain how to set up a python virtual environment that can be used to benchmark the accuracy and runtime of different fitting minimizers in Mantid for different fitting problems. To do this, we will use a package called `FitBenchmarking <https://fitbenchmarking.readthedocs.io/en/stable/>`_ which is a cross-platform open source tool for comparing different minimizers and fitting frameworks.
+
+Initial Setup
+#############
+
+Some initial setup is required on your system before we can start setting up an environment for benchmarking Mantid minimizers. Before you start this process, make sure you have a recent version of python installed and access to a terminal or command prompt.
+
+1. Download and install the desired version of Mantid from the `downloads <https://download.mantidproject.org/>`_ page (if not already installed). Note its installation path, hereby denoted as ``[INSTALL_PATH]``.
+2. Open a terminal or command prompt and cd to the desired location.
+3. Pip install the virtual environment package as described in the 'Installing virtualenv' section found `here <https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#installing-virtualenv>`_.
+
+Creating your Benchmarking Environment
+######################################
+
+1. From the same terminal or command prompt, run the following command:
+
+.. code-block:: sh
+
+  py -m virtualenv -p=[INSTALL_PATH]\bin\python.exe benchmark-env
+
+This will create a virtual environment based on the python executable provided with your Mantid installation. For Windows, the location of this executable is ``C:\MantidInstall\bin\python.exe``. In this example, ``benchmark-env`` is the name given to your virtual environment.
+
+2. Activate your environment. This is explained in the 'Activating a virtual environment' section found `here <https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#activating-a-virtual-environment>`_.
+3. Install the FitBenchmarking package by following the instructions found `here <https://fitbenchmarking.readthedocs.io/en/stable/users/install_instructions/fitbenchmarking.html>`_.
+
+Your environment should now be ready for performing a benchmark of Mantid minimizers.
+
+Running a Benchmark
+###################
+
+The process for how to run a benchmark is explained extensively in the `FitBenchmarking documentation <https://fitbenchmarking.readthedocs.io/en/stable/users/index.html>`_, and so I recommend you give it a read. This section will give a basic example of how to perform a simple benchmark of three Mantid minimizers.
+
+1. Create a file called ``fitting_options.ini`` with the following contents
+
+.. code-block:: text
+
+  [FITTING]
+
+  software: mantid
+
+  num_runs: 1
+  max_runtime: 600
+
+  [MINIMIZERS]
+
+  mantid: Levenberg-Marquardt
+          Levenberg-MarquardtMD
+          Simplex
+
+  [PLOTTING]
+
+  make_plots: yes
+
+  [OUTPUT]
```

Comment:
I had to comment out the whole [OUTPUT] section in order to run fitbenchmarking successfully.